tests/html/cloned-template-script.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<!--
+    @license
+    Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <title>cloned template script Test</title>
+    <script src="../../../template/template.js"></script>
+    <script src="../../html-imports.min.js"></script>
+    <script>WCT = {waitFor: function(cb){cb()}};</script>
+    <script src="../../../web-component-tester/browser.js"></script>
+    <link rel="import" href="imports/template-import.html" id="template-script">
+  </head>
+  <body>
+    <script>
+      test('cloned template script execution', function(done) {
+        // In native HTMLImports the event 'HTMLImportsLoaded' might have already
+        // been fired by this time, so use HTMLImports.whenReady.
+        HTMLImports.whenReady(function() {
+          chai.assert.isUndefined(window.executedTemplateScript, 'remote template script is NOT executed after import');
+          chai.assert.isUndefined(window.executedNestedTemplateScript, 'remote nested template script is NOT executed after import');
+          const template = document.querySelector('#template-script').import.querySelector('template');
+          document.head.appendChild(document.importNode(template.content, true));
+          chai.assert.isTrue(window.executedTemplateScript, 'remote template script executed after clone and attach to the main document');
+          chai.assert.isUndefined(window.executedNestedTemplateScript, 'remote nested template script is NOT executed after parent is clone and attach to the main document');
+          const nestedTemplate = document.querySelector('#nested-template')
+          document.head.appendChild(document.importNode(nestedTemplate.content, true));
+          chai.assert.isTrue(window.executedNestedTemplateScript, 'remote nested template script executed after clone and attach to the main document');
+          done();
```

Comment:
It would be good to add a spy or some other count to verify that template scripts are executed only once per stamping.

Reply:
Something like `assert.isTrue(window.executedNestedTemplateScript === 1)`, with import that does `window.executedNestedTemplateScript++`?

Reply:
Yep, I agree we should check that the script is executed only once! I've added more tests in https://github.com/webcomponents/html-imports/pull/77/commits/26f0463fa8a9e9312a2b49e0ec47c9868d4c158a